[Synthetics] Adjust private location docs to support base elastic-agent (#3831)
elastic/docs-content: reference/fleet/elastic-agent-container.md

### Complete flavor

> If you want to run Synthetics Browser tests, run the docker pull command to fetch the elastic-agent-complete image:

Reviewer: ```suggestion
If you want to run Synthetics Browser tests and Lightweight tests, run the docker pull command to fetch the elastic-agent-complete image:
```

### Complete flavor

In case you need to run {{agent}} as a Synthetic Private Location and you'll use only Lightweight monitors, you can use the `elastic-agent` (and the `wolfi` variant).

>  In case you need to run {{agent}} as a Synthetic Private Location and you'll use both Lightweight monitors and Synthetics Browser tests, run the docker pull command to fetch the `elastic-agent-complete` image:

Reviewer 2: ```suggestion
To run {{agent}} as a Synthetic Private Location using both Lightweight monitors and Synthetics Browser tests, run the following docker pull command to fetch the `elastic-agent-complete` image:
```

1. Set to 1 to enroll the {{agent}} into {{fleet-server}}.
2. URL to enroll the {{fleet-server}} into. You can find it in {{kib}}. Select **Management → {{fleet}} → Fleet Settings**, and copy the {{fleet-server}} host URL.
3. The token to use for enrollment. Close the flyout panel and select **Enrollment tokens**. Find the Agent policy you want to enroll {{agent}} into, and display and copy the secret token. To learn how to create a policy, refer to [Create an agent policy without using the UI](/reference/fleet/create-policy-no-ui.md).

> 4. Switch `elastic-agent` to `elastic-agent-complete` if you intend to use the complete version. Use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetic tests cannot run under the root user. Refer to [Synthetics {{fleet}} Quickstart](/solutions/observability/synthetics/get-started.md) for more information.

Reviewer 3: ```suggestion
4. Switch `elastic-agent` to `elastic-agent-complete` if you intend to use the complete version. Use the `elastic-agent` user instead of root to run Synthetic browser monitors. Synthetic browser monitors cannot run under the root user. Refer to [Synthetics {{fleet}} Quickstart](/solutions/observability/synthetics/get-started.md) for more information.
```

3. The {{fleet}} service token. [Generate one in the {{fleet}} UI](/reference/fleet/fleet-enrollment-tokens.md#create-fleet-enrollment-tokens) if you don’t have one already.
4. ID of the {{fleet-server}} policy. We recommend only having one fleet-server policy. To learn how to create a policy, refer to [Create an agent policy without using the UI](/reference/fleet/create-policy-no-ui.md).
5. publish container port 8220 to host.

> 6. Switch `elastic-agent` to `elastic-agent-complete` if you intend to use the complete version. Use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetic tests cannot run under the root user. Refer to [Synthetics {{fleet}} Quickstart](/solutions/observability/synthetics/get-started.md) for more information.

Reviewer 3: ```suggestion
6. Switch `elastic-agent` to `elastic-agent-complete` if you intend to use the complete version. Synthetic browser monitors cannot run under the root user ‒ use the `elastic-agent` user instead. Refer to [Synthetics {{fleet}} Quickstart](/solutions/observability/synthetics/get-started.md) for more information.
```
